requires_test.txt

```diff
@@ -4,4 +4,4 @@ mygene ==3.2.2
 scvi-tools >=1.1.0, <1.3.0
 dgl >=1.1.0, <1.2.0
 torch_geometric ==2.5.3
-torch ==2.1.1
+torch ==2.7.1
```

Comment:
```suggestion
torch ==2.5.1
```
we wil need to switch to to latest version with https://github.com/scverse/scvi-tools
dues to:
```
tests/test_model_server.py:192: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tdc/model_server/tdc_hf.py:73: in load
    model.load()
tdc/model_server/models/scvi.py:54: in load
    self.model = scvi_package.model.SCVI.load('scvi_model', adata)
.venv/lib/python3.9/site-packages/scvi/model/base/_base_model.py:681: in load
    ) = _load_saved_files(
.venv/lib/python3.9/site-packages/scvi/model/base/_utils.py:66: in _load_saved_files
    model = torch.load(model_path, map_location=map_location)
```